spv/envelope_test.go

```diff
@@ -1,33 +0,0 @@
-package spv
```

Comment:
what's the reason why we removed the envelope_tests?

Reply:
Accidental deletion - the test it runs isn't really relevant to binary format, but we should keep it anyway - no reason to delete.